circle/docker-release-image.sh

```diff
@@ -285,6 +285,24 @@ chart_update_image() {
   esac
 }
 
+chart_update_appVersion() {
```

Comment:
This function we determine the appVersion from the image tag. I'm not sure how we should deal with charts that consist two or more images, fe. the [parse chart](https://github.com/bitnami/charts/blob/master/incubator/parse/values.yaml) consists of parseServer and parseDashboard images, each having a different version, and updating each of these will trigger a separate chart update. 

Suggestion are welcome /cc @prydonius